src/citrine/resources/ara_definition.py

```diff
@@ -228,6 +229,52 @@ def build(self, data: dict) -> AraDefinition:
         defn.session = self.session
         return defn
 
+    def build_ara_table(self, ara_def: AraDefinition) -> JobSubmissionResponse:
+        """[ALPHA] submit an ara table construction job.
+
+        This method makes a call out to the job framework to start a new job to build
+        the respective table for a given AraDefinition.
+
+        Parameters
+        ----------
+        project: Project
+            The project on behalf of which the job executes.
+        ara_def: AraDefinition
+            the ara definition describing the new table
+
+        """
+        job_id = uuid4()
+        url_suffix: str = "/{ara_definition}/versions/{version_number}/build?job_id={job_id}"
```

Comment:
typically we use `params=...` in the call to `post_resource` instead of embedding them in the URL directly.